examples/output/EMDataset-001.json

```diff
@@ -0,0 +1,133 @@
+{
+  "acquisition": {
+    "detector": "Falcon 4i",
+    "dose_per_movie": {
+      "unit": "e/Å²",
```

Comment:
Did we decide whether to use `electrons/Å^2` or `Å^-2` for this?

Reply:
we decided on Å^-2 as electrons are not a unit and to avoid confusion with the scicat SI converter. The fact the count concerns electrons is explicitly specified in the schemas field description.